cost-analyzer/values.yaml

```diff
@@ -103,10 +103,6 @@ kubecostModel:
     #  cpu: "800m"
     #  memory: "256Mi"
 
-kubecostInit:
```

Comment:
is there any way this can cause errors, i.e. someone still passing this value?

if not then, LGTM

Reply:
Passing extraneous values is fine.